using-kubecost/navigating-the-kubecost-ui/efficiency.md

```diff
@@ -0,0 +1,45 @@
+# Efficiency Report
+
+Efficiency Report is a Monitoring dashboard which provides workload and infra idle costs for your clusters. Metrics include consumption cost of individual CPU/RAM/storage metrics, and total workload efficiency.
+
+Efficiency Report supports clusters by all major cloud service providers, and on-prem clusters.
+
+![Efficiency Report](/images/efficiency.png)
+
+## Understanding effiency and idle cost metrics
+
+Efficiency Report breaks down idle costs into two distinct categories:
+
+* *Workload idle* refers to hardware resources (CPU/GPU/RAM) that are requested by the workloads, but are not consumed on average.
+* *Infra idle* refers to purchased hardware resources that are neither requested nor consumed.
+
+These metrics, when compared to total spend, are necessary when calculating efficiency. Workload efficiency can be calculated as:
+
+> ((resources consumed/resources requested) * 100)
+
+For more information about efficiency and idle metrics, see our [Efficiency and Idle](/using-kubecost/navigating-the-kubecost-ui/cost-allocation/efficiency-idle.md) doc.
+
+## Configuring your query
+
+### Date Range
+
+Select the date range of the report by setting specific start and end dates, or using one of the preset options.
+
+### Aggregate By
+
+Here you can aggregate your cost metrics by either *Cluster* or *Node*. Hovering your cursor over indiviudal graph items will display a legend breaking down each color with the cost of the resource associated with it. The sum of all cost metrics in your window will equal the Total Cost metric displayed in the below table.
+
+### Filters
+
+Filter your results at the cluster or node level, where only matching resources will be displayed. Supports advanced filtering options as well.
+
+### Idle
+
+One of the most important tools for configuring your query is the idle dropdown. Here, you can choose which idle costs to display, but this will also affect the reported cost metrics in the Efficiency Report table:
+
+* *Idle by type*: Will display table columns for both workload idle and infra idle, and provides workload efficiency percentage

```

Comment:
I think this will have both Cluster Efficiency and Workload Efficiency in the end.